airflow/www/views.py

```diff
@@ -1293,50 +1302,72 @@ def set_duration(tid):
     def graph(self, session=None):
         dag_id = request.args.get('dag_id')
         blur = conf.getboolean('webserver', 'demo_mode')
-        dag = dagbag.get_dag(dag_id)
-        if dag_id not in dagbag.dags:
-            flash('DAG "{0}" seems to be missing.'.format(dag_id), "error")
-            return redirect('/')
-
         root = request.args.get('root')
+        read_from_file = request.args.get('read_from_file')
```

Comment:
does the button still exist or not?

Reply:
The button is gone but the functionality is still in there as hidden argument. This is used at the point there is not a dag run yet for a dag. In this case flask will redirect to `/graph?dag_id=dag_name&read_from_file=True`. This has no harm on the user because they don't even know this happens ;)